### PR TITLE
Getting carousel to work.

### PR DESCRIPTION
**Pull request by dwilliamson5**

### CODE REVIEW

File: public/css/main.css [hunk 1]
```diff
@@ -40,6 +40,16 @@ footer {
   height: 236px;
 }
 
+.carousel_image {
+  width: 100%;
+  height: 450px;
+}
+
+.carousel-caption h3 {
+  color:crimson;
```
Comment (dwilliamson5) [inline public/css/main.css:50]: Do we want the description on top of the image? If so, what color do you make it? Some pictures are black vs white

File: views/items/show.handlebars [hunk 2]
```diff
@@ -22,15 +22,21 @@
...
+          <img class="carousel_image" src="{{this.imageUrl}}" alt="{{this.description}}">
+        
+          <div class="carousel-caption d-none d-md-block">
+            <h3>{{this.description}}</h3>
+          </div>
+        </div>
+      {{/each}}
+
+    {{else}}
+      <p>There are no images yet. <a href="/items/{{itemId}}/editPhotos" class="btn btn-primary">Click here</a> to add!</p>
```
Comment (matttarantino) [inline views/items/show.handlebars]: Only the item owner should see this

Comment (matttarantino) [reply views/items/show.handlebars]: For a regular user, I think we can something like "Ask the owner in the comments to add photos"

File: public/css/main.css (hunk 1 above)
Comment (matttarantino) [reply public/css/main.css:50]: @dwilliamson5 this might make things easier: https://getbootstrap.com/docs/4.0/components/carousel/#with-captions

Comment (dwilliamson5) [reply public/css/main.css:50]: Thank you. I did follow that but the color still seems to be an issue depending on the photo

**Pull request merged**